### PR TITLE
Switch to using TLOG_ERRS_RAISE macros in .m4 files

### PR DESCRIPTION
**Pull request by danghai**
Hi Nick, PR for switching the code to use error handling macro in `*.m4` file.

### CODE REVIEW

File: lib/tlog/rec_session_conf_cmd.c.m4 [hunk 1]
```diff
@@ -121,14 +113,13 @@ tlog_rec_session_conf_cmd_load(struct tlog_errs **perrs,
...
-                            "of the program name");
-            goto cleanup;
+            TLOG_ERRS_RAISECS(grc,
+                              "Failed allocating a copy of the shell part "
+                              "of the program name");
         }
         /* Decode and unescape the shell path */
-        for (dst = src = shell; ;) {
+        for (dst = src = shell;
+                          ;) {
```
Comment (spbnick) [inline lib/tlog/rec_session_conf_cmd.c.m4]: Could you please not change this line?

File: m4/tlog/conf_cmd.m4 [hunk 2]
```diff
@@ -310,7 +310,7 @@ m4_dnl
 m4_define(
     `M4_CONF_CMD_HELP_OPTS',
     `m4_pushdef(`m4_orig_divnum', m4_divnum)m4_divert(-1)
-        m4_pushdef(`M4_CONTAINER', m4_defn(`M4_CONF_CMD_HELP_OPTS_CONTAINER')), 
+        m4_pushdef(`M4_CONTAINER', m4_defn(`M4_CONF_CMD_HELP_OPTS_CONTAINER')),
```
Comment (spbnick) [inline m4/tlog/conf_cmd.m4]: Could you please make this a separate change, or not change it at all?

**Comment by spbnick** [comment]
Ah, another thing: could you please change the commit subject to something like "Switch using TLOG_ERRS_RAISE macros in .m4 files"? This way it is more specific, doesn't mention "code" which is a bit redundant, and also doesn't have a shell expansion character `*` in it, which can be troublesome in some cases.

File: m4/tlog/conf_cmd.m4 (hunk 2 above)
Comment (danghai) [reply m4/tlog/conf_cmd.m4]: Oh It is weird. I do not change anything in this line.

**Comment by spbnick** [comment]
Thanks a lot, Hai, merged!

**Pull request merged**